### PR TITLE
feat(api): new hunting license client

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

New client for the hunting license from Umhverfisstofnun

## Why

So we can display the license on Mínar Síður

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

File: apps/api/infra/api.ts [hunk 1]
```diff
@@ -222,6 +223,11 @@ export const serviceSetup = (services: {
...
         prod: 'https://innskra.island.is',
       },
+      HUNTING_LICENSE_PASS_TEMPLATE_ID: {
+        dev: '1da72d52-a93a-4d0f-8463-1933a2bd210b',
+        staging: '1da72d52-a93a-4d0f-8463-1933a2bd210b',
+        prod: 'd4ecf781-3764-4063-a4e1-9c3e17cebfba',
+      },
       XROAD_RSK_PROCURING_REDIS_NODES: {
         dev: json([
           'clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com:6379',
```
Comment (busla) [inline apps/api/infra/api.ts:233]:
```suggestion
          'redis-applications.internal:6379',
```

**Review by busla — CHANGES_REQUESTED** [review]

File: apps/api/infra/api.ts (hunk 1 above)
Comment (thorkellmani) [reply apps/api/infra/api.ts:233]: dont know that this does but sure

**Review by busla — APPROVED** [review]
🙌🏼

**Review by jonarnarbriem — APPROVED** [review]

**Review by thordurhhh — APPROVED** [review]
Code owned files looking good 🚀

File: libs/auth-api-lib/seeders/data/client-licenses-ust.ts
Comment (saevarma) [inline libs/auth-api-lib/seeders/data/client-licenses-ust.ts:1]: Then you need to talk to Jón Arnar to generate and get the client's secret 

cc @jonarnarbriem

File: libs/clients/license-client/src/lib/clients/hunting-license-client/huntingLicenseClient.service.ts [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+  Result,
+  VerifyPkPassResult,
+} from '../../licenseClient.type'
+import {
+  HuntingLicenseClientService,
+  HuntingLicenseDto,
+} from '@island.is/clients/hunting-license'
+
+/** Category to attach each log message to */
+const LOG_CATEGORY = 'adrlicense-service'
```
Comment (saevarma) [inline libs/clients/license-client/src/lib/clients/hunting-license-client/huntingLicenseClient.service.ts:27]: adr?

**Review by saevarma — APPROVED** [review]
LGTM

File: libs/auth-api-lib/seeders/data/client-licenses-ust.ts
Comment (thorkellmani) [reply libs/auth-api-lib/seeders/data/client-licenses-ust.ts:1]: yep, we're in cahoots

File: libs/clients/license-client/src/lib/clients/hunting-license-client/huntingLicenseClient.service.ts (hunk 2 above)
Comment (thorkellmani) [reply libs/clients/license-client/src/lib/clients/hunting-license-client/huntingLicenseClient.service.ts:27]: whoops 😵‍💫

**Pull request merged**